developers/weaviate/configuration/persistence.md

```diff
@@ -22,11 +22,12 @@ services:
       - /var/weaviate:/var/lib/weaviate
     environment:
       CLUSTER_HOSTNAME: 'node1'
+      PERSISTENCE_DATA_PATH: '/var/lib/weaviate'
 ```
 
 * About the volumes
   * `/var/weaviate` is the location where you want to store the data on the local machine
-  * `/var/lib/weaviate` (after the colon) is the location inside the container, don't change this
+  * `/var/lib/weaviate` (after the colon) is the location inside the container, don't change this as it needs to match whatever is in `PERSISTENCE_DATA_PATH` variable.
```

Comment:
@dudanogueira is it ok to update the location in both places? 

If so, consider:
 
The value after the colon is the storage location inside the container. This value must match the `PERSISTENCE_DATA_PATH` variable.